src/logistration/LoginPage.jsx

```diff
@@ -121,6 +124,11 @@ class LoginPage extends React.Component {
     });
   }
 
+  handleCreateAccountLinkClickEvent() {
+    sendGAEvent('analytics', 'edx.bi.register_form.toggled', { category: 'user-engagement' });
+    sendGAEvent('page', 'login_and_registration register');
```

Comment:
Same for the register, I'm not sure but I guess `register` should go as type in params? It's going as a separate param in old implementation https://github.com/edx/edx-platform/blob/5a09786cf3547a5d6ea395d4f7522c04efcb0a5f/lms/static/js/student_account/views/AccessView.js#L288.

Reply:
I think i was hardcoding the generated string ie pagename + type like **login_and_registration login**.  I am not sure how and for what purpose type is used in backbonejs and how they are combined to create above event name . My assumption is it is treated as category. https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/#page
I have changed the code to reflect this now